### ISSUE TITLE
Not updated weight backbone_3d

### ISSUE THREAD

**Issue by csj777** [WoodwindHu/DTS#10]
when I run test for your pretrained model: nusc-kitti-pvrcnn, nusc-kitti-secondiou. I get error: 
```
2023-09-09 14:14:00,499   INFO  Not updated weight backbone_3d.conv_input.0.weight: torch.Size([3, 3, 3, 4, 16])
2023-09-09 14:14:00,499   INFO  Not updated weight backbone_3d.conv1.0.0.weight: torch.Size([3, 3, 3, 16, 16])
2023-09-09 14:14:00,499   INFO  Not updated weight backbone_3d.conv2.0.0.weight: torch.Size([3, 3, 3, 16, 32])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv2.1.0.weight: torch.Size([3, 3, 3, 32, 32])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv2.2.0.weight: torch.Size([3, 3, 3, 32, 32])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv3.0.0.weight: torch.Size([3, 3, 3, 32, 64])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv3.1.0.weight: torch.Size([3, 3, 3, 64, 64])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv3.2.0.weight: torch.Size([3, 3, 3, 64, 64])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv4.0.0.weight: torch.Size([3, 3, 3, 64, 64])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv4.1.0.weight: torch.Size([3, 3, 3, 64, 64])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv4.2.0.weight: torch.Size([3, 3, 3, 64, 64])
2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv_out.0.weight: torch.Size([3, 1, 1, 64, 128])
2023-09-09 14:14:00,504   INFO  ==> Done (loaded 177/189)
```
I guess it may be due to the version of spconv, but I'm not sure.  I'm using spconv1.2

**Comment by WoodwindHu** [comment]
torch==1.10.1
spconv==2.1.21

**Comment by maxiuw** [comment]
the issue here is (imo) that when starting the self training you add the DSNorm layer. However, when training sourceonly models you only use batch norm layers. That causes this issue. There is a function which ST3D++ also uses to translate the model to DSNorm model but it does not seem to work properly. You can try setting DSNorm as False in Self train part of your config and this error shoud disappear, however, this doesn not seem like the best solution.

**Comment by ZSA010807** [comment]
> when I run test for your pretrained model: nusc-kitti-pvrcnn, nusc-kitti-secondiou. I get error:当我对您的预训练模型进行测试时：nusc-kitti-pvrcnn、nusc-kitti-secondiou。我收到错误：
> 
> ```
> 2023-09-09 14:14:00,499   INFO  Not updated weight backbone_3d.conv_input.0.weight: torch.Size([3, 3, 3, 4, 16])
> 2023-09-09 14:14:00,499   INFO  Not updated weight backbone_3d.conv1.0.0.weight: torch.Size([3, 3, 3, 16, 16])
> 2023-09-09 14:14:00,499   INFO  Not updated weight backbone_3d.conv2.0.0.weight: torch.Size([3, 3, 3, 16, 32])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv2.1.0.weight: torch.Size([3, 3, 3, 32, 32])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv2.2.0.weight: torch.Size([3, 3, 3, 32, 32])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv3.0.0.weight: torch.Size([3, 3, 3, 32, 64])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv3.1.0.weight: torch.Size([3, 3, 3, 64, 64])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv3.2.0.weight: torch.Size([3, 3, 3, 64, 64])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv4.0.0.weight: torch.Size([3, 3, 3, 64, 64])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv4.1.0.weight: torch.Size([3, 3, 3, 64, 64])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv4.2.0.weight: torch.Size([3, 3, 3, 64, 64])
> 2023-09-09 14:14:00,504   INFO  Not updated weight backbone_3d.conv_out.0.weight: torch.Size([3, 1, 1, 64, 128])
> 2023-09-09 14:14:00,504   INFO  ==> Done (loaded 177/189)
> ```
> 
> I guess it may be due to the version of spconv, but I'm not sure. I'm using spconv1.2我猜这可能是由于 spconv 的版本所致，但我不确定。我正在使用 spconv1.2

I encountered the same issue. Have you managed to resolve it?
